Source/ResourceLib/Accelerator.cs

```diff
@@ -65,6 +98,26 @@ public UInt32 Command
         }
 
         /// <summary>
+        /// Adds a single flag to the Accelerator bitflags
+        /// </summary>
+        /// <param name="f">the User32.AcceleratorVirtualKey flag to add</param>
+        public void addFlag(User32.AcceleratorVirtualKey f)
+        {
+            if (!((_accel.fVirt & (uint)f) == (uint)f))

```

Comment:
You can set this via a `|=`, see http://stackoverflow.com/questions/93744/most-common-c-sharp-bitwise-operations-on-enums for various options of how to implement that.